Gruntfile.js

```diff
@@ -6,7 +6,10 @@ module.exports = function(grunt) {
 			options: {
 				configFile: ".eslintrc.json"
 			},
-			target: ["js/*.js", "modules/default/*.js", "serveronly/*.js", "*.js"]
+			target: ["js/*.js", "modules/default/*.js", "modules/default/*/*.js",
+				"serveronly/*.js", "*.js", "!modules/default/alert/notificationFx.js",
+				"!modules/default/alert/modernizr.custom.js", "!modules/default/alert/classie.js"
```

Comment:
I like the idea of this, but it would be better to put these into the .eslintignore file in the root of the project.